recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
+
+# Set fixed IP for eth2
+SRC_URI += "file://eth2.connection"
+
+PACKAGECONFIG_remove = "ifupdown dnsmasq"
+
+# configure networkmanager with modemmanager support
+PACKAGECONFIG_append = " modemmanager ppp"
```

Comment:
Consider using `:=`syntax. This eliminates possible problems with missing spaces.

Reply:
I don't see whats wrong with this?

Reply:
Sorry, of course.

Reply:
Changed to +=